instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/RuntimeMetricsBuilder.java

```diff
@@ -22,6 +22,7 @@ public final class RuntimeMetricsBuilder {
 
   private boolean disableJmx = false;
   private boolean enableExperimentalJmxTelemetry = false;
+  private boolean enableCaptureGcCause = false;
```

Comment:
rename to `captureGcCause `

Reply:
I followed the similar naming format with `enableExperimentalJmxTelemetry` and `disableJmx` here. If use `captureGcCause`, it seems to cause difference format in variable naming.

Reply:
I tend to agree with @laurit that 

`otel.instrumentation.runtime-telemetry.capture-gc-cause` maps well to `captureGcCause` (and the `enable` is redundant, and there's no `.enable` in the property name)

`otel.instrumentation.runtime-telemetry.emit-experimental-telemetry` probably maps better to `emitExperimentalTelemetry` for similar reason if you want to change that too and deprecate the old method name.